templates/midonet.repo.j2

```diff
@@ -0,0 +1,20 @@
+[{{ midonet_repo_repository_section_prefix }}]
+name={{ midonet_repo_repository_description_prefix }}
+baseurl={{ midonet_repo_repository_base_url }}/{{ midonet_repo_midonet_dir }}/{{ midonet_repo_midonet_release|default('stable') }}/el{{ ansible_distribution_major_version }}/
+enabled=1
+gpgcheck=1
+gpgkey={{ midonet_repo_repository_gpgkey_url }}
+
+[{{ midonet_repo_repository_section_prefix }}-openstack-integration]
+name={{ midonet_repo_repository_description_prefix }} OpenStack Integration
+baseurl={{ midonet_repo_repository_base_url }}/openstack-{{ midonet_repo_openstack_plugin_version }}/{{ midonet_repo_openstack_plugin_release|default('stable') }}/el{{ ansible_distribution_major_version }}/
+enabled=1
+gpgcheck=1
+gpgkey={{ midonet_repo_repository_gpgkey_url }}
+
+[{{ midonet_repo_repository_section_prefix }}-misc]
```

Comment:
there is no MEM or non-nem misc. repo, but only one thing, name it [midonet-misc]
same goes for descriptions